appinfo/routes.php

```diff
@@ -74,6 +74,11 @@
 			'url' => '/api/accounts/{accountId}/folders/{folderId}/stats',
 			'verb' => 'GET'
 		],
+		[
+			'name' => 'folders#subscribeFolder',
+			'url' => '/api/accounts/{accountId}/folders/{folderId}/subscribeFolder',
```

Comment:
```suggestion
			'url' => '/api/accounts/{accountId}/folders/{folderId}/subscription',
```

This is a bit cosmetic/idealistic. But REST routes should be about resources, not actions. This goes more into that direction in my opinion. What do you think?

Reply:
Sounds reasonable, will change it.